lib/screens/profile/faq.dart

```diff
@@ -19,92 +25,57 @@ class FAQItem {
   });
 }
 
-class FAQScreen extends StatefulWidget {
+class FAQScreen extends HookConsumerWidget {
   const FAQScreen({super.key});
 
   @override
-  State<FAQScreen> createState() => _FAQScreenState();
-}
+  Widget build(BuildContext context, WidgetRef ref) {
+    final categories = useState(ref.read(gqlClientProvider).query$FAQ());
```

Comment:
This will query the api for every build 👯‍♂️ 
You need to use useMemoized(() => )

Reply:
And for the naming here, I think we should have a convention from now on:

final somethingFuture = useMemoized(() => fetch()); // Future<T>
final somethingSnapshot = useFuture(somethingFuture) // AsyncSnapshot<T>

Reply:
> This will query the api for every build 👯‍♂️ You need to use useMemoized(() => )

Oh yea. It didn't occur to me.😱